test/e2e/operator_test.go

```diff
@@ -56,6 +60,33 @@ func getClient() (client.Client, string, error) {
 	return kubeClient, namespace, nil
 }
 
+func newClusterIngress(name, ns, domain string) *ingressv1alpha1.ClusterIngress {
+	ci := &ingressv1alpha1.ClusterIngress{
```

Comment:
Could just `return &ingressv1alpha1.ClusterIngress{  ... }`  as `ci` is not really used locally.

Reply:
Good catch. I had a few diff variations of the func and forgot to simply return the ptr on my final iteration. Thx!